pkgs/applications/video/gnome-subtitles/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, fetchFromGitLab
+, autoreconfHook
+, pkg-config
+, intltool
+, itstool
+, mono
+, yelp-tools
+, wrapGAppsHook
+, enchant1
+, glib
+, gnome3
+, gtk-sharp-3_0
+, gst_all_1
+, libxml2
+}:
+let
+  version = "1.6";
+in
+stdenv.mkDerivation {
+  pname = "gnome-subtitles";
+  inherit version;
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-subtitles";
+    rev = "gnome-subtitles-${version}";
+    sha256 = "1crmmcx32i6ca7dlr3xhnc7vgv9jhlpwh6hxhv2fl1x1zbasf42z";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    mono
+    intltool
+    itstool
+    pkg-config
+    yelp-tools
+  ];
+
+  buildInputs = [
+    enchant1
+    gnome3.gtk
+    libxml2
+    glib
+    gtk-sharp-3_0
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-plugins-base
+    (gst-plugins-good.override {
+      gtkSupport = true;
+    })
+    gst-plugins-bad
+  ]);
+
+  preConfigure = ''
+    intltoolize
+  '';
+
+  preFixup =
+    let
+      libPath = stdenv.lib.makeLibraryPath [
+        enchant1
+        glib
+        gnome3.gtk
+      ];
+    in
+    ''
+      gappsWrapperArgs+=(
+        --prefix MONO_PATH : "${gtk-sharp-3_0}/lib/mono/gtk-sharp-3.0"
+        --prefix LD_LIBRARY_PATH : "${libPath}"
```

Comment:
I am not fond of setting `LD_LIBRARY_PATH` since it carries over to child processes, which can cause problems but I have not idea if there is a better way for Mono.

Reply:
Maybe try absolute paths in target?

Reply:
Still not found a way to avoid using `LD_LIBRARY_PATH`. But the present code works. An the update from 1.6 to 1.7.2 made the packaging a little easier (not having to patch files).